src/main/java/seedu/address/model/good/UniqueGoodList.java

```diff
@@ -95,6 +98,12 @@ public void setGoods(List<Good> goods) {
         }
 
         internalList.setAll(goods);
+        sort();
+    }
+
+    private void sort() {
+        internalList.sort((a, b) ->
+                a.quantityLowerThanThreshold() ? -1 : b.quantityLowerThanThreshold() ? 1 : -1);
```

Comment:
Can you add comment to this ternary operators to explain it?

Reply:
ok